scene/main/node.cpp

```diff
@@ -1302,6 +1302,13 @@ Node::AutoTranslateMode Node::get_auto_translate_mode() const {
 
 bool Node::can_auto_translate() const {
 	ERR_READ_THREAD_GUARD_V(false);
+
+#ifdef TOOLS_ENABLED
+	if ((Engine::get_singleton()->is_editor_hint() || Engine::get_singleton()->is_project_manager_hint()) && !is_part_of_edited_scene()) {
+		return false;
+	}
+#endif
```

Comment:
We do need auto translation in the editor.

As mentioned in https://github.com/godotengine/godot/pull/96105#issuecomment-2310925588, to solve double translation, it's better to change most `TTR` calls to `TTRC`.

`ETR`s are for engine strings that could be included in the user generated POT files, like strings in file dialogs and the color picker.

Reply:
Indeed, switching to `TTRC` is probably better, disabling `atr` seems to disable some of the inspector translations.